data/101/815/075/101815075.geojson

```diff
@@ -100,6 +103,9 @@
     "name:und_x_variant":[
         "Totsi"
     ],
+    "name:unk_x_variant":[
+        "Totsi"
```

Comment:
unk and und 🤷 ?

Reply:
https://github.com/whosonfirst-data/whosonfirst-data-admin-ee/pull/11#discussion_r463332361, same applies here... but this is interesting because there shouldn't be a `unk` name property anymore. I'll investigate.

Reply:
Fixed in b9e8726.